fractions/keycloak/src/main/java/org/wildfly/swarm/keycloak/runtime/SecuredArchivePreparer.java

```diff
@@ -100,6 +128,9 @@ private Asset createAsset(InputStream in) {
     }
 
     @Configurable("swarm.keycloak.json.path")
-    private String keycloakJsonPath;
+    String keycloakJsonPath;
+
+    @Configurable("swarm.keycloak.security-constraints")
```

Comment:
Not sure, but wondering if `security-constraints` should be `security.constraints`?